woodwork/type_sys/utils.py

```diff
@@ -113,7 +113,9 @@ def list_semantic_tags():
     tags_df = tags_df.append(
         pd.DataFrame([['index', False, [ww.type_system.str_to_logical_type(tag) for tag in ['integer', 'double', 'categorical', 'datetime']]],
                       ['time_index', False, [ww.type_system.str_to_logical_type('datetime')]],
-                      ['date_of_birth', False, [ww.type_system.str_to_logical_type('datetime')]]
+                      ['date_of_birth', False, [ww.type_system.str_to_logical_type('datetime')]],
+                      ['ignore', False, 'Any LogicalType'],
+                      ['passthrough', False, 'Any LogicalType']
```

Comment:
Adding a string makes it clearer that all logical types are valid than listing all logical types would be, but means that we couldn't use the values to, say, select by logical type for columns where these tags would be permissible (not sure that that's even an expected use case...)